src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+/** Association class to support the following functionality:
+ * When the AddressBook program exits, it should print out a list of all the tags added/deleted during that session.
+ */
+
+public class Tagging {
+	private ArrayList<String> allTheTags = new ArrayList<>();
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class. If it was in this class, the diagram would be like this:
![image](https://cloud.githubusercontent.com/assets/1673303/18409597/69111e4e-777d-11e6-92bf-9914c825bd13.png)